examples/mlflow/keras_mlflow.py

```diff
@@ -0,0 +1,96 @@
+"""
+Optuna example that optimizes a neural network regressor for the
+wine quality dataset using Keras and records hyperparamters and metrics using MLflow.
+
+In this example, we optimize the learning rate and momentum of
+stochastic gradient descent optimizer to minimize the validation mean squared error
+for the wine quality regression.
+
+You can run this example as follows:
+    $ python keras_mlflow.py
+
```

Comment:
[question] Is it possible to execute this example from CLI like other examples?

e.g.
https://github.com/optuna/optuna/blob/04baada1970b6a78f97a866a3d7da7afccc56d4e/examples/keras_simple.py#L8-L18